structure_tests/build.sh

```diff
@@ -26,4 +26,4 @@ fi
 
 envsubst < cloudbuild.yaml.in > cloudbuild.yaml
 cd ..
-gcloud alpha container builds create . --config=structure_tests/cloudbuild.yaml
+gcloud beta container builds submit . --config=structure_tests/cloudbuild.yaml
```

Comment:
I had originally kept this as `alpha` because some machines we run some of these jobs on don't keep gcloud up to date. should be fine as long as we monitor the machines we run this build job on.